Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/README.md

```diff
@@ -0,0 +1,490 @@
+Azure Risky Users provides access to all at-risk users and risk detections in Azure AD environment.
+This integration was integrated and tested with version 1.0 of Microsoft Graph AzureRiskyUsers.
```

Comment:
They call the product `AzureRiskyUsers`?With no spaces?

Reply:
Done